app/build.gradle.kts

```diff
@@ -33,10 +33,17 @@ android {
         }
     }
     buildTypes {
-        debug { isMinifyEnabled = false }
-        release {
-            isMinifyEnabled = true
-            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+        // for local builds, used to find shrinking issues
+        val isMinify = project.hasProperty("minify")
```

Comment:
I added this parameter / property to my global gradle.properties to always have shrinking enabled.

Reply:
Allright :)